### DISCUSSION TITLE
Unable to migrate from Windows app with 81character seed

### DISCUSSION THREAD

**Discussion by javierceballosaltuna** [iotaledger/firefly#8625] [Help & Support]
Hi, just downloaded latest version of firefly app and "Migrate from legacy" option just redirects me to: https://blog.iota.org/iota-legacy-migration-tool/

No way to proceed this way, any help is appreciated

**Comment by begonaalvarezd** [comment]
Thats a tutorial on how to migrate using a standalone app called "Legacy Migration Tool", follow the steps described in that tutorial, you cant migrate using Firefly
